server/routerlicious/packages/routerlicious-base/src/alfred/runnerFactory.ts

```diff
@@ -167,21 +169,64 @@ export class AlfredResourcesFactory implements utils.IResourcesFactory<AlfredRes
 
         const tenantManager = new services.TenantManager(authEndpoint);
 
+        // Rest API Throttler
         const throttleMaxRequestsPerMs =
             config.get("alfred:throttling:maxRequestsPerMs") as number | undefined;
         const throttleMaxRequestBurst =
             config.get("alfred:throttling:maxRequestBurst") as number | undefined;
-        const throttleMinCooldownIntervalInMs =
-            config.get("alfred:throttling:minCooldownIntervalInMs") as number | undefined;
-        const minThrottleIntervalInMs =
-            config.get("alfred:throttling:minThrottleIntervalInMs") as number | undefined;
+        const throttleMinRequestCooldownIntervalInMs =
+            config.get("alfred:throttling:minRequestCooldownIntervalInMs") as number | undefined;
+        const throttleMinRequestThrottleIntervalInMs =
+            config.get("alfred:throttling:minRequestThrottleIntervalInMs") as number | undefined;
         const throttleStorageManager = new services.RedisThrottleStorageManager(redisClient);
         const restThrottlerHelper = new services.ThrottlerHelper(
             throttleStorageManager,
             throttleMaxRequestsPerMs,
             throttleMaxRequestBurst,
-            throttleMinCooldownIntervalInMs);
-        const restThrottler = new services.Throttler(restThrottlerHelper, minThrottleIntervalInMs, winston);
+            throttleMinRequestCooldownIntervalInMs);
+        const restThrottler = new services.Throttler(
+            restThrottlerHelper,
+            throttleMinRequestThrottleIntervalInMs,
+            winston);
+
+        // Socket Connection Throttler
+        const throttleMaxSocketConnectionsPerMs =
+            config.get("alfred:throttling:maxSocketConnectionsPerMs") as number | undefined;
```

Comment:
let's add default values instead of undefined?

Reply:
The default values are defined in [Throttler](https://github.com/microsoft/FluidFramework/blob/main/server/routerlicious/packages/services/src/throttler.ts#L27) and [ThrottlerHelper](https://github.com/microsoft/FluidFramework/blob/main/server/routerlicious/packages/services/src/throttlerHelper.ts#L19-L21). `undefined` here is actually a type declaration acknowledging that `nconf` will return `undefined` if the config value does not exist.